languages/scxml/src/test/java/de/jplag/scxml/ScxmlParserTest.java

```diff
@@ -0,0 +1,120 @@
+package de.jplag.scxml;
+
+import static de.jplag.scxml.parser.model.executable_content.SimpleExecutableContent.Type.ASSIGNMENT;
+import static de.jplag.scxml.parser.model.executable_content.SimpleExecutableContent.Type.SCRIPT;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.xml.parsers.ParserConfigurationException;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.xml.sax.SAXException;
+
+import de.jplag.ParsingException;
+import de.jplag.scxml.parser.ScxmlParser;
+import de.jplag.scxml.parser.model.State;
+import de.jplag.scxml.parser.model.Statechart;
+import de.jplag.scxml.parser.model.Transition;
+import de.jplag.scxml.parser.model.executable_content.*;
+import de.jplag.scxml.util.StateBuilder;
+
+class ScxmlParserTest {
```

Comment:
Just in case you generate files in some tests, use clean-up methods to delete them. See the model instance PR, for example.